bin/stacks/routing-database-stack.ts

```diff
@@ -5,8 +5,22 @@ import { Construct } from 'constructs'
 
 export interface RoutingDatabaseStackProps extends cdk.NestedStackProps {}
 
+export const V3PoolsDynamoDbTableName = 'V3PoolsCachingDB'
+export const V3PoolsDynamoDbPartitionKeyName = 'poolAddress'
+export const V3PoolsDynamoDbSortKeyName = 'blockNumber'
+export const TTLAttributeName = 'ttl'
+
+export const V3PoolsDynamoDbProps: aws_dynamodb.TableProps = {
```

Comment:
I would prefer that this configuration is inline with the Dynamo Table creation, it feels like a layer of indirection for it to be here.

Reply:
> This will not yet populate the table, it is just creating the AWS instance, right?

We clarified offline. "populate the table" = provision the table, but I agree populate sounds more like writing entries into the table.